Upgrade to ipfs@0.41 (#381)
ProtoSchool/protoschool.github.io: src/tutorials/0005-regular-files-api/04-exercise.md

Using the `cat` function, retrieve the file's contents and return them as a string.

> **Hint:** The `CID` we provide is a string, so it needs to be placed inside quotes. Also, don't forget to convert the buffered contents of the text file to a string using `.toString()` before returning your result.

Reviewer: Join the hints!

Using the `cat` function, retrieve the file's contents and return them as a string.

> **Hint:** The `CID` we provide is a string, and as such it be placed inside quotes. Also, don't forget to convert the buffered contents of the text file to a string using `.toString()` before returning your result.

Reviewer: ```suggestion
**Hint:** The `CID` we provide is a string, so it needs to be placed inside quotes. Also, don't forget to convert the buffered contents of the text file to a string using `.toString()` before returning your result.
```
